### PR TITLE
Add webperf label to existing web performance related automated tests

### PR DESCRIPTION
**Pull request by davehunt**
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our README.md: https://github.com/web-platform-tests/wpt-metadata/blob/master/README.md 
2. Please label this pull request `do not merge yet` upon creation because the auto-merge feature is enabled in this repository. Once your PR is created, do not enable auto-merge on it. If this pull request is finished, feel free to assign foolip/kyleju as reviewers, or we will review and merge them automatically.

### CODE REVIEW

**Comment by davehunt** [comment]
> 2. Please label this pull request `do not merge yet` upon creation because the auto-merge feature is enabled in this repository. Once your PR is created, do not enable auto-merge on it. If this pull request is finished, feel free to assign foolip/kyleju as reviewers, or we will review and merge them automatically.

I wasn't able to add a label to the pull request, but I have marked it as a draft.

**Comment by davehunt** [comment]
This was raised in the [WebPerfWG call - November 21st 2024](https://docs.google.com/document/d/13JhDQ89yQcKgrivQfvX-TDKWMN-rq5x_PcgPcbNFGx0/edit?tab=t.0#heading=h.fqb3bceg91h7). Request for opinions or changes was made. There was a positive response, and no concerns were raised during the call, and as far as I can tell there have been none since. I believe this is ready for review.

File: beacon/META.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 links:
+    - label: webperf
+      results:
+        - test: *
```
Comment (jgraham) [inline beacon/META.yml:4]: Looks like all the existing metadata ends up quoting `*` like `test: '*'`. I'm not sure if that avoids some YAML pitfall, but I"m not sure it doesn't, so let's be consistent.

File: device-memory/META.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+links:
+    - label: webperf
+      results:
+        - test: *
```
Comment (jgraham) [inline device-memory/META.yml:4]: Fix the missing newlines.

**Comment by davehunt** [comment]
Thanks @jgraham I have addressed your review comments.

**Review by jgraham — APPROVED** [review]

**Pull request merged**